spec/models/train_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+
+describe Train do
+  it { should have_valid(:name).when("Super Line") }
```

Comment:
Great! This is just what I was looking for. Please delete lines 11 - 40, which are redundant.